hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/util/TokenHandlerHelper.java

```diff
@@ -321,25 +306,14 @@ public static TokenRelation getIfUsable(
             @NonNull final AccountID accountId,
             @NonNull final TokenID tokenId,
             @NonNull final ReadableTokenRelationStore tokenRelStore) {
-        return getIfUsable(accountId, tokenId, tokenRelStore, TokenRelValidations.REQUIRE_NOT_FROZEN);
-    }
-
-    @NonNull
-    public static TokenRelation getIfUsable(
-            @NonNull final AccountID accountId,
-            @NonNull final TokenID tokenId,
-            @NonNull final ReadableTokenRelationStore tokenRelStore,
-            @NonNull final TokenRelValidations tokenRelValidations) {
         requireNonNull(accountId);
         requireNonNull(tokenId);
         requireNonNull(tokenRelStore);
 
         final var tokenRel = tokenRelStore.get(accountId, tokenId);
 
         validateTrue(tokenRel != null, TOKEN_NOT_ASSOCIATED_TO_ACCOUNT);
-        if (tokenRelValidations == TokenRelValidations.REQUIRE_NOT_FROZEN) {
-            validateTrue(!tokenRel.frozen(), ACCOUNT_FROZEN_FOR_TOKEN);
-        }
+        validateTrue(!tokenRel.frozen(), ACCOUNT_FROZEN_FOR_TOKEN);
```

Comment:
Son't we need to not check this for tokenReject ?

Reply:
The default case is that it checks if the account is frozen. This logic is redundant since it was decided that for TokenReject we won't have to bypass the frozen status